tests/unit/test_download.py

```diff
@@ -65,6 +69,58 @@ def read(self, *args, **kwargs):
         self._count += 1
 
 
+class TestDownloadFilenameOutputManager(BaseTaskTest):
+    def setUp(self):
+        super(TestDownloadFilenameOutputManager, self).setUp()
+        self.osutil = OSUtils()
+        self.download_output_manager = DownloadFilenameOutputManager(
+            self.osutil, self.transfer_coordinator)
+
+        # Create a file to write to
+        self.tempdir = tempfile.mkdtemp()
+        self.filename = os.path.join(self.tempdir, 'myfile')
+
+        self.call_args = CallArgs(fileobj=self.filename)
+        self.future = self.get_transfer_future(self.call_args)
+
+    def tearDown(self):
+        super(TestDownloadFilenameOutputManager, self).tearDown()
+        shutil.rmtree(self.tempdir)
+
+    def test_is_compatible(self):
```

Comment:
Could we also test for something that's incompatible, like a `StringIO` or something.